uaclient/config.py

```diff
@@ -526,6 +526,16 @@ def status(self, show_beta=False) -> "Dict[str, Any]":
 
         Write the status-cache when called by root.
         """
+
+        # Try to remove fix reboot notices if not applicable
+        if not util.should_reboot():
+            self.remove_notice(
+                "",
+                status.MESSAGE_ENABLE_REBOOT_REQUIRED_TMPL.format(
+                    operation="fix operation"
```

Comment:
I think we probably want to clear all reboot required notices in this case right? If so, let's take advantage of globbing in the descr_regex param
```suggestion
                    operation=".*"
```

Reply:
No, I don't think we can't do this because of FIPS PRO case. After we upgrade the package in a FIPS PRO machine, we will emit a notice telling that the user needs to restart the system to install the right metapackage. However, we will not have a `/var/run/reboot-required` file in that situation. This means that if the user runs `ua status` after upgrading the package in that machine, we will remove the notice.

Because of that, I am only removing the `fix` notices on that situation.